### ISSUE TITLE
Chinese characters not supported in preediting popup

### ISSUE THREAD

**Issue by PositivePaulo** [uim/uim#150]
It seems I can't get the preediting popup to display Chinese characters correctly. I think it might simply be a matter of font selection or installation, but I don't know which is used by the popup. I have some Chinese fonts installed on my system, but that doesn't do the trick.

Installation details:
* UIM 1.8.8-4
* UIM-QT5 1.8.8-4
* UIM-Pinyin 1.8.8-4

System environment:
* OS: Debian 4.19.67-2+deb10u2, kernel version 4.19.0-6-amd64
* Desktop environment: KDE 5

**Comment by kou** [comment]
Could you show a screenshot for the problem?

**Comment by PositivePaulo** [comment]
@kou Yes of course, sorry. I had trouble getting one yesterday because Kate would keep crashing when switching to another window while currently preediting (it might be another problem?), but I now included a screenshot in the original post.
